pages/userprofile.js

```diff
@@ -1,10 +1,70 @@
 import SignedInLayout from '../views/layouts/signedInLayout';
-import TrackCard from '../components/TrackCard';
+import CardBackground from '../views/cardBackground';
 
+const name = "bob";
+
+const UserDetails = (props) => (
+    <div id="user-detail-div">
+        <div className="card-body">
+            <div className="row">
+                <div className="col-md-6">
+                    <h5 className="card-title text-success">Name: {props.userName}</h5>
+                </div>
+                <div className="col-md-6">
+                    <button id="userNameButton" className="btn btn-outline-dark text-center">Change Name</button>
```

Comment:
Change id to hyphen format.  Change to **user-name-button** for HTML.  Create clear distinction from variables and JSX for React.